lib/linguist/languages.yml

```diff
@@ -4013,6 +4013,12 @@ Processing:
   tm_scope: source.processing
   ace_mode: text
   language_id: 294
+Proguard:
+  type: programming
+  color: "#0096D8"
+  extensions:
+  - ".pro"
+  language_id: 716513858
```

Comment:
You'll need the following too:
```yaml
tm_scope: none
ace_mode: text
```
The grammar is not mandatory.

Reply:
Done, thanks!